examples/simple-game-server/Makefile

```diff
@@ -35,7 +35,7 @@ WITH_WINDOWS=1
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
-server_tag = $(REGISTRY)/simple-game-server:0.10
+server_tag = $(REGISTRY)/simple-game-server:0.11
```

Comment:
This PR needs a new simple-game-server because of the https://github.com/googleforgames/agones/pull/2489.
But, I am not sure this will trigger the build pipeline or not.